qualtran/bloqs/arithmetic/addition.py

```diff
@@ -266,6 +268,120 @@ def _add_oop_large() -> OutOfPlaceAdder:
 )
 
 
+@frozen
+class SimpleAddConstant(Bloq):
+    r"""Applies U_{k}|x> = |x + k>.
```

Comment:
I personally don't find it useful to have this `U` in all these different expressions. Suggest something like:

```
"""Takes |x> to |x + k> for a classical integer `k`
```